pwnlib/util/misc.py

```diff
@@ -394,7 +394,7 @@ def register_sizes(regs, in_sizes):
          'ch': ['ecx', 'cx', 'ch'],
          'cl': ['ecx', 'cx', 'cl'],
          'cx': ['ecx', 'cx'],
-         'dh': ['edx', 'dx', 'dh'],
+       'dh': ['edx', 'dx', 'dh'],
```

Comment:
Whoa! Ugly accidental whitespace changes
```suggestion
         'dh': ['edx', 'dx', 'dh'],
```